libnmstate/ifaces/ifaces.py

```diff
@@ -165,6 +166,35 @@ def _validate_ovs_patch_peers(self):
                             " patch port"
                         )
 
+    def _validate_vlan_mtu_lt_base_mtu(self):
+        for iface in self._ifaces.values():
```

Comment:
Should check whether `iface.mtu` is valid first.

Reply:
Should I raise NmStateValueError if mtu of vlan is not present? Or just skip the check then